.changeset/polite-spiders-pay.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-auth-backend': minor
+'@backstage/plugin-auth-node': minor
+---
+
+Added configurable algorithm field for IdentityClient and TokenFactory
```

Comment:
Could you split this into two changesets, one for each package? Both packages are on 0.x and this isn't a breaking change, so should be `patch` bumps as well.